docs/en/v0.4/summary.yml

```diff
@@ -65,8 +65,8 @@
     - kubernetes
     - gtctl
     - run-on-android
+    - monitor-&-alert
```

Comment:
this character is dangerous as in file system path and url, consider removing it.

Reply:
oh, let me change this name